spec/support/fixtures/integrations/discord/interactions/ping_interaction.rb

```diff
@@ -0,0 +1,55 @@
+require 'faker'
+require 'json'
+
+# rubocop: disable Metrics/ParameterLists
+# rubocop: disable Metrics/CyclomaticComplexity
+# rubocop: disable Metrics/PerceivedComplexity
+module Integrations
+  module Discord
+    module Interactions
+      class PingInteraction
+        TYPE = 1
```

Comment:
this is duplicated. Pls use the hash you defined on the Request body or whatever replacement you decide to use for it if the Request class is gone

Reply:
fixtures are now removed